backend/chainlit/cli/__init__.py

```diff
@@ -7,6 +7,7 @@
 
 nest_asyncio.apply()
 
+# ruff: noqa: E402
```

Comment:
Not sure if it is necessary to call `nest_asyncio.apply()` before the other imports.

Reply:
Feel free to add this literally as a comment. @willydouhard Happy to hear from you why this is here, if you've a clue.

Reply:
I added the comment.